code/Modules/Items/Carriables/Item.cs

```diff
@@ -13,15 +13,18 @@ namespace ssl.Modules.Items.Carriables
     /// </summary>
     public partial class Item : Carriable, ISelectable
     {
+        public const string Tag = "Item";
```

Comment:
**suggestion**: change name to ItemTag
When you are here you know you are on item tag.